packages/terra-button/src/Button.jsx

```diff
@@ -115,221 +115,194 @@ const defaultProps = {
   variant: ButtonVariants.NEUTRAL,
 };
 
-class Button extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { active: false, focused: false };
-    this.handleKeyDown = this.handleKeyDown.bind(this);
-    this.handleKeyUp = this.handleKeyUp.bind(this);
-    this.handleOnBlur = this.handleOnBlur.bind(this);
-    this.handleFocus = this.handleFocus.bind(this);
-    this.handleMouseDown = this.handleMouseDown.bind(this);
-    this.handleClick = this.handleClick.bind(this);
-
-    this.shouldShowFocus = true;
```

Comment:
How are we accounting for this shouldShowFocus variable in the new logic?

Reply:
Discussed in sync: Un-necessary variable now removed.